app/helpers/plants_helper.rb

```diff
@@ -12,7 +12,7 @@ def plant_img(plant)
   def ppl_have
     render_haml <<-HAML
       .text-2xl.py-1.font-semibold.rounded.bg-green-200
-        %span= GardenPlant.where(plant_id: @plant.id).count
+        %span= @plant.users.count
```

Comment:
The GardenPlant is a join table, it's only job is to enable us call has_many on users and plants. besides if I don't know what GardenPlant was, I would have no idea what is happening here hence we use `plant.users`